hpx/runtime/applier/trigger.hpp

```diff
@@ -10,6 +10,7 @@
 
 #include <hpx/runtime/actions/continuation.hpp>
 
+/*
```

Comment:
Yes, this was removed intentionally. the whole file could go as those functions aren't used anywhere.